tests/unit/utils/patch-splitter-test.js

```diff
@@ -1,13 +1,76 @@
+var patch = [
+  'diff -Nr -U 1000 -x \'*.png\' generated/v5.1.0/Gemfile generated/v5.1.1/Gemfile',
+  '--- generated/v5.1.0/Gemfile\t2017-05-13 00:39:57.000000000 +0000',
+  '+++ generated/v5.1.1/Gemfile\t2017-05-13 00:39:57.000000000 +0000',
+  '@@ -1,54 +1,54 @@',
+  ' source \'https://rubygems.org\'',
+  '',
+  ' git_source(:github) do |repo_name|',
+  '   repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")',
+  '   "https://github.com/#{repo_name}.git"',
+  ' end',
+  '',
+  '',
+  ' # Bundle edge Rails instead: gem \'rails\', github: \'rails/rails\'',
+  '-gem \'rails\', \'~> 5.1.0\'',
+  '+gem \'rails\', \'~> 5.1.1\'',
+  ' # Use sqlite3 as the database for Active Record',
+  ' gem \'sqlite3\'',
+  ' # Use Puma as the app server',
+  ' gem \'puma\', \'~> 3.7\'',
+  ' # Use SCSS for stylesheets',
+  ' gem \'sass-rails\', \'~> 5.0\'',
+  ' # Use Uglifier as compressor for JavaScript assets',
+  ' gem \'uglifier\', \'>= 1.3.0\'',
+  ' # See https://github.com/rails/execjs#readme for more supported runtimes',
+  ' # gem \'therubyracer\', platforms: :ruby',
+  '',
+  ' # Use CoffeeScript for .coffee assets and views',
+  ' gem \'coffee-rails\', \'~> 4.2\'',
+  ' # Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks',
+  ' gem \'turbolinks\', \'~> 5\'',
+  ' # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder',
+  ' gem \'jbuilder\', \'~> 2.5\'',
+  ' # Use Redis adapter to run Action Cable in production',
+  ' # gem \'redis\', \'~> 3.0\'',
+  ' # Use ActiveModel has_secure_password',
+  ' # gem \'bcrypt\', \'~> 3.1.7\'',
+  '',
+  ' # Use Capistrano for deployment',
+  ' # gem \'capistrano-rails\', group: :development',
+  '',
+  ' group :development, :test do',
+  '   # Call \'byebug\' anywhere in the code to stop execution and get a debugger console',
+  '   gem \'byebug\', platforms: [:mri, :mingw, :x64_mingw]',
+  '   # Adds support for Capybara system testing and selenium driver',
+  '-  gem \'capybara\', \'~> 2.13.0\'',
+  '+  gem \'capybara\', \'~> 2.13\'',
+  '   gem \'selenium-webdriver\'',
+  ' end',
+  '',
+  ' group :development do',
+  '   # Access an IRB console on exception pages or by using <%= console %> anywhere in the code.',
+  '   gem \'web-console\', \'>= 3.3.0\'',
+  '   gem \'listen\', \'>= 3.0.5\', \'< 3.2\'',
+  '   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring',
+  '   gem \'spring\'',
+  '   gem \'spring-watcher-listen\', \'~> 2.0.0\'',
+  ' end',
+  '',
+  ' # Windows does not include zoneinfo files, so bundle the tzinfo-data gem',
+  ' gem \'tzinfo-data\', platforms: [:mingw, :mswin, :x64_mingw, :jruby]',
+  '',
+].join("\n");
```

Comment:
I know you made this change to fix the test (thanks for that), but it's important that the content of the diff be about multiple files, otherwise we're not really testing the splitting of diffs. The failing test has now been fixed in `master`.